deploy/chart/values.yaml

```diff
@@ -53,6 +53,8 @@ package:
     internalPort: 5443
   nodeSelector:
     kubernetes.io/os: linux
+  securityContext: 
+    runAsUser: 1001
```

Comment:
```suggestion
```
Same

Reply:
The image `quay.io/operator-framework/olm:master` must define the USER ID instead

Reply:
@anik120 We need to define the user in : https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Dockerfile#L36-L38